services/statestorage/merkle/merkel_test.go

```diff
@@ -0,0 +1,12 @@
+package merkle
```

Comment:
1. Notice that the name "merkle" is spelled in different two ways in this path

2. If the Merkle libarary will be usable by other code in the project (like Merkle root of the transactions in a block), maybe we should move it from /services/statestorage to /crypto